core/src/main/java/org/elasticsearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -54,6 +61,24 @@ public DisMaxQueryBuilder add(QueryBuilder queryBuilder) {
     }
 
     /**
+     * Add a list of sub-queries to this disjunction.
+     */
+    public DisMaxQueryBuilder add(List<QueryBuilder> queryBuilders) {
```

Comment:
Just as a convenience to not have to loop over all queries to add in the parser. We have similar methods in BoolQueryBuilder. I can ommit the loop here and user addAll() I guess.